hirop_msgs/JointData[] joints
---
int8 result